apollo.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   client: {
     service: {
       name: 'easi-app',
-      url: process.env.REACT_APP_GRAPHQL_ADDRESS
+      localSchemaFile: './pkg/graph/schema.graphql'
```

Comment:
This should allow the Apollo extension to work even when the server isn't running.